### DISCUSSION TITLE
Download Links

### DISCUSSION THREAD

**Discussion by a842322** [XFox111/TabsAsideExtension#78] [Q&A]
Hi there, I am wondering if it is possible to download all of the links from my extension into a HTML file of some sorts so that I can port between accounts?

**Comment by XFox111** [comment]
@a842322, Hi. In the future we plan to add an ability to export your sets to browser bookmarks, where you can save them to a file. There's a 
#74 PR for the feature by @AmineI. You can track it or even contribute
